release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -742,6 +742,10 @@ The Local Storage Operator now has the ability to:
 
 For more information, see xref:../storage/persistent_storage/persistent-storage-local.adoc#local-storage-discovery_persistent-storage-local[Automating discovery and provisioning for local storage devices].
 
+[id="ocp-4-6-aws-efs-tp-removed-1st"]
+==== External provisioner for AWS EFS (Technology Preview) feature has been removed
+The Amazon Web Services (AWS) Elastic File System (EFS) technology preview feature has been removed and is no longer supported.
+
```

Comment:
We have claimed that EFS was removed in the 4.5 release note,  in 4.6, we marked it in `Table 1. Deprecated and removed features tracker` as `REM` should be enough?

Reply:
In this chapter, looks like we only doc the new changes in this version.

Reply:
I talked to Jan about this and we agreed that since we are retroactively removing this feature from all releases, rather than actually removing it starting with 4.5, that we should include the removal notice in all versions of the rel notes. Make sense?

Reply:
Got it.